h/traversal/roots.py

```diff
@@ -75,7 +75,14 @@
 from h.traversal import contexts
 
 
-class Root:
+class RootFactory:
+    """Abstract interface for root resource factories."""
```

Comment:
```suggestion
    """Base class for all root resource factories."""
```

I'm not sure I understand in what sense this is an abstract interface (it doesn't specify any abstract methods in either an informal or a formal ABCs way). I think it's just a base class

Reply:
I suppose in the sense that this isn't an actual root in the app (distinct from `Root`).